.travis.yml

```diff
@@ -2,5 +2,8 @@ language: node_js
 node_js:
   - "6"
 script:
-- python populateFeaturesFiles.py
-- npm run eslint
+  - python generateFeedChanges.py
```

Comment:
Although, now that I've read this the whole way through, I think it might just be there for legacy feature files, yeah?